### PR TITLE
frontend: extend account info with P2TR

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "verify": "Verify on device",
     "xpubTypeChangeBtn": {
       "p2pkh": "View legacy P2PKH extended public key",
-      "p2wpkh": "View Native segwit",
+      "p2tr": "View taproot",
+      "p2wpkh": "View native segwit",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json:38]: i changed this on locize already

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:38]: LGTM, but I'd always use capital `T` for the name `Taproot`

**Review by thisconnect — APPROVED** [review]
LGTM with suggestion

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (benma) [reply frontends/web/src/locales/en/app.json:38]: :+1: done

**Pull request merged**